winpr/libwinpr/sspi/Negotiate/negotiate.c

```diff
@@ -96,6 +96,7 @@ static const WinPrAsn1_OID negoex_OID = { 10, (BYTE*)"\x2b\x06\x01\x04\x01\x82\x
 
 #ifdef WITH_KRB5
 static const SecPkg SecPkgTable[] = {
+	{ KERBEROS_SSP_NAME, &KERBEROS_SecurityFunctionTableA, &KERBEROS_SecurityFunctionTableW },
```

Comment:
Looks like a duplicate

Reply:
@mfleisz yes, but look at the struct array below ;)